ipyvolume/_version.py

```diff
@@ -3,4 +3,6 @@
 __version__ = '0.5.2-dev.1'
 __version_tuple_js__ = (0, 5, 2, 'dev.1')
 __version_js__ = '0.5.2-dev.1'
-__version_threejs__ = '0.91' # kept for embedding in offline mode, we don't care about the patch version since it should be compatible
\ No newline at end of file
+__version_threejs__ = '2.0.1'
```

Comment:
threejs is at version 98 btw: https://threejs.org/
pythreejs is what you meant, I know, it is confusing.
I'll talk with @vidartf about it, how we can get the right threejs version that matches the pythreejs one.